### DISCUSSION TITLE
Appliance images unavailable

### DISCUSSION THREAD

**Discussion by Maintech3** [Security-Onion-Solutions/securityonion#5113] [General] [answered]
I have no idea where to begin to troubleshoot this issue. It is showing up in Grid. I updated and this started. I'm not even sure what files to look at or paste here for you to look at. This new S.O. is above my pay-grade I am afraid. If anyone knows where to begin I would be very grateful. Hopefully, at the most, all you would need to do is post a link or two that will guide me in learning this new system and guide me in the repair. Thank you in advance.

**Comment by TOoSmOotH** [comment]
Appliance images only show up for Security Onion appliances.

**Comment by Maintech3** [comment, marked as answer]
I do understand they only show up for Security Onion appliances. But my point was that I never had that issue before. So, I guess I had Security Onion Appliances :<. I have somewhat repaired it though not completely. I have managed to get the "!" removed from the grid icon. Now it says "Status OK" but at the same time it still says "Appliance images unavailable." And it says:
Online Since: Aug 9, 2021 1:41 PM
Production EPS: 2
Consumption EPS: 2
Process Status: OK
Connection Status: OK
Raid Status: Unknown
This was after a reboot. Correct me if I am wrong but the images are datasets. Right?

**Reply by TOoSmOotH** [reply]
If you have appliances please contact support and they can fix it for you.

**Reply by dougburks** [reply]
> Correct me if I am wrong but the images are datasets. Right?

The images are pictures of the front and rear of the official Security Onion Solutions appliances as shown here:
https://docs.securityonion.net/en/2.3/grid.html

**Reply by Maintech3** [reply]
My apologies. My time has been limited and I have not spent the time learning the new S.O. as I should have. I will not bother the forum again until I have studied all online tutorials and any other self learning educational materials you offer. Also, I knew that appliances were the hardware in connected nodes but had spent too many hours troubleshooting networking, system OS, you name it and then, in the end, the S.O. I'm not making excuses. It was my fault and I apologize for taking up your time needlessly.
